BedrockServer.cpp

```diff
@@ -638,10 +645,11 @@ void BedrockServer::worker(SData& args,
                     // We need to have multi-write enabled, the command needs to not be explicitly blacklisted, and it
                     // needs to not be automatically blacklisted.
                     canWriteParallel = canWriteParallel && BedrockConflictMetrics::multiWriteOK(command.request.methodLine);
-                    if (!canWriteParallel               ||
-                        state != SQLiteNode::MASTERING  ||
-                        command.httpsRequest            ||
-                        command.onlyProcessOnSyncThread ||
+                    if (!canWriteParallel                 ||
+                        server._suppressMultiWrite.load() ||
```

Comment:
NAB: Tiny thing, but I'd suggest putting the most expensive and locking call at the end of the condition, such that it's called less frequently.